### DISCUSSION TITLE
[Feature] Allow private playlists

### DISCUSSION THREAD

**Discussion by gtntdev** [mediacms-io/mediacms#612] [Feature Requests]
You can't add private playlists (private videos to playlists) and you can't share videos between users (private)
These features (especially the first one) would be nice to have but thanks for the great work you did so far!

Best regards,
Lukas

PS: if you want me to delete the second part of the request, just tell me:)

**Comment by mgogoulos** [comment]
This is valid, private media are password protected - this needs to be exposed on the frontend and then they will be available to be shared or added on playlists.

**Comment by u35253** [comment]
One highly related request, to expand upon this, would be to please allow Unlisted videos to be added to (Unlisted) playlists.  I believe that an Unlisted property for playlists would be a new concept.

Currently, it is not possible to create a playlist by starting from an Unlisted video, or containing Unlisted videos.

But this would be useful still.  As a logged in user, I would want to be able to upload Unlisted videos, add them to an Unlisted Playlist, and share the Unlisted Playlist for a logged-out user to be able to watch.

I understand that the semantics and edge cases could get a little counterintuitive, but overall this would still be a very useful feature IMO.  Please consider it requested in this discussion.

**Comment by LaserKaspar** [comment]
Hey, this seems to be related: https://github.com/mediacms-io/mediacms/discussions/1248
Is it also the case that you can't add unlisted videos to a playlist?
